### PR TITLE
Add support for format time

### PR DESCRIPTION
**Pull request by radeno**
Add support for time type.

I am aware that time format looks like normal string separated by colon, but in the future we should be able to do some transformations if needed. So for now, let's support simple implementation.

Same with the Duration format, But it will be in different PR.

### CODE REVIEW

File: core/package.json [hunk 1]
```diff
@@ -49,7 +49,7 @@
     "webpack-cli": "^4.10.0"
   },
   "engines": {
-    "node": "16.x.x",
```
Comment (radeno) [inline core/package.json:52]: It works fine on Node v18

File: core/src/templates/template.fo.xslt.js [hunk 2]
```diff
@@ -24,7 +24,7 @@ export default `<?xml version="1.0" encoding="utf-8"?>
     <xsl:param name="values" />
 
     <xsl:choose>
-
```
Comment (radeno) [inline core/src/templates/template.fo.xslt.js:27]: prettier

File: core/src/templates/template.html.xslt.js [hunk 3]
```diff
@@ -135,6 +135,11 @@ export default `<?xml version="1.0" encoding="utf-8" standalone="yes"?>
     <xsl:value-of select="concat($dd,'.',$mm,'.',$yyyy)"/>
   </xsl:template>
 
+  <xsl:template name="base_format_time">
+    <xsl:param name="instr"/>
+    <xsl:value-of select="format-time($instr, '[H01]:[m01]')"/>
```
Comment (radeno) [inline core/src/templates/template.html.xslt.js:140]: Format time with leading zero for now to be backward compatible with simple string.

**Review by mpinter — APPROVED** [review]
:+1: makes sense, lgtm. And thanks for the update.

**Pull request merged**